docs/testing.md

```diff
@@ -0,0 +1,78 @@
+CRI-Containerd Testing Guide
+============================
+This document assumes you have already setup the development environment (go, git, `cri-contaienrd` repo etc.).  
+ 
+Before sending pull requests you should at least make sure your changes have passed code verification, unit test and CRI validation test.
+## Code verification
```

Comment:
We call it [`source code verification`](https://github.com/kubernetes-incubator/cri-containerd/blob/master/Makefile#L40) in Makefile, too.

If we generate a TOC in the future, I think `Code Verification` will be better than `Verification Test`. :)